### PR TITLE
Webumenia 1117 add vimeo to item

### PR DESCRIPTION
**Pull request by igor-kamil**

### CODE REVIEW

File: resources/views/vendor/backpack/crud/fields/items_sortable.blade.php [hunk 1]
```diff
@@ -95,7 +100,35 @@
...
                      e.preventDefault();
                      $('#items  > option:selected').each(function() {
-                         $('#table > tbody').append('<tr class="array-row item"><td>' + $(this).text() + '</td><td>' + $(this).data('type') + '</td><td><input class="sorter form-control" size="2" data-id="' + $(this).val() + '" value="0" name="items_sort[' + $(this).val() + ']"></td><td><span class="btn btn-sm btn-light sort-handle pull-right ui-sortable-handle"><span class="sr-only">sort item</span><i class="fa fa-sort" role="presentation" aria-hidden="true"></i></span></td><td><button data-id="' + $(this).val() + '" class="btn btn-sm btn-light removeItem" type="button"><span class="sr-only">delete item</span><i class="fa fa-trash" role="presentation" aria-hidden="true"></i></button></td></tr>');
+
+                        var preview_img = '';
+                        if ($(this).data('preview')) {
+                          preview_img = `<img src="${ $(this).data('preview') }" alt="" style="height: 50px; width: auto">`;
+                        }
+
+                        var table_row = `
```
Comment (eronisko) [inline resources/views/vendor/backpack/crud/fields/items_sortable.blade.php]: I'll try but I'm afraid this won't work on IE :(
Quick fix would be to replace `${}` with some placeholders and then do a simple string replace. Or better, using a Vue component

Comment (igor-kamil) [reply resources/views/vendor/backpack/crud/fields/items_sortable.blade.php]: could you please test it?

Comment (eronisko) [reply resources/views/vendor/backpack/crud/fields/items_sortable.blade.php]: Sorry, we obviously don't care about IE here if it's just admin.

Comment (igor-kamil) [reply resources/views/vendor/backpack/crud/fields/items_sortable.blade.php]: meanwhile I fixed that :D

**Pull request merged**